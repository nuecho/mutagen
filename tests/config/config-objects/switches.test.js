importDependencies("switch-dependencies-config.json");
cfgObjectTests("switches", "switches-config.json", null, 0, false);
