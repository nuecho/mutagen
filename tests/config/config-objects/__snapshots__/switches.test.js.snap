// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen switches config import-export] should create the switches: the switches have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ switch => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen switches config import-export] should throw exception when new switches to import miss mandatory properties switches: exception has been thrown 1`] = `
Object {
  "stderr": "Mandatory properties [physicalSwitch] not set in switch 'Environment/emptyCfgObjectTest'
Cannot import configuration: some configuration objects' mandatory properties for creation are not set.
",
  "stdout": "",
}
`;

exports[`[mutagen switches config import-export] should update the switches: the switches have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ switch => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;
